### DISCUSSION TITLE
[@types/cloudflare] EnterpriseZoneWorkersKV.addMulti has wrong type parameters

### DISCUSSION THREAD

**Discussion by asgeirn** [DefinitelyTyped/DefinitelyTyped#61813] [Issues with a @types package] [answered]
The `data` object has wrong signature:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/cloudflare/index.d.ts#L114-L117

Compare this to the API:

https://github.com/cloudflare/node-cloudflare/blob/master/lib/resources/EnterpriseZoneWorkersKV.js#L95-L104

See https://api.cloudflare.com/#workers-kv-namespace-write-multiple-key-value-pairs for actual parameters.

I defined this type:

```
type KeyValue = {
    key: string
    value: string
    expiration?: number
    expiration_ttl?: number
    metadata?: object
    base64?: boolean
}
```

[a comment by an automated account is not shown]

**Comment by samchouse** [comment, marked as answer]
Can confirm, can you submit a PR just replacing the current object type with that one? Thanks!

**Reply by asgeirn** [reply]
Sure thing! #61858
